src/Products/urban/browser/exportimport/import_config.py

```
@@ -166,7 +166,7 @@ def get_uid_from_proximity_context(self, context, id, ignore_uid=[]):
         return self.get_uid_from_proximity_context(aq_parent(context), id, ignore_uid)
 
     def get_template_uid(self, item, template):
-        if isinstance(template["template"], str):
+        if isinstance(template["template"], basestring):
```

Ruff (0.15.13)

[error] Undefined name `basestring`

(F821)